paper/paper.md

```diff
@@ -0,0 +1,482 @@
+each session to answer questions and help learners debug.
+Furthermore, one or two facilitators were assigned to each project group to help
+learners define data analysis questions, develop and execute a data analysis
+plan, visualize and communicate their findings, and troubleshoot coding
+problems. The culmination of the project is a presentation to peers, facilitators,
+and family members. Through this process learners gain hands-on experience
+coding, cleaning data, performing statistical analyses, creating informative data
+visualizations, and communicating their results to others.
+Projects have ranged from investigating exoplanets to studying the
+genomics of psoriasis.
+
+We credit the success of our curriculum not only to the skill of the
+instructors, but also to the way we organized and executed the lessons and
+project:
+
+1. The instructors and learners used [Google Colaboratory
+(Colab)](https://colab.research.google.com/) to write and execute code in
+Jupyter notebooks. We chose this option because learners do not have to install
+any programs to use Google Colab, and you can easily open and edit the Jupyter
+notebooks from GitHub.  When meeting in person, most learners use Google
+Chromebooks which have limited programming capabilities, but easy use of a web
+browser.
+1. Assigning facilitators to groups allowed learners to build a more personal
+connection with their facilitators, making them feel more comfortable asking
+questions.
+1. Group projects were performed using paired programming to allow learners to
+collaborate and learn from each other.
+1. We used the "sticky note" system from The Carpentries by which learners can
+ask for help by putting up a colored sticky note (or a Zoom emoji in the case of
+virtual meetings) [@becker_responding_2016].
+1. We exposed the learners to different aspects of data science by bringing in
+women guest speakers from academics and industry. This allowed them to better
+put what they were learning into context, think about how they might use the
+skills they were learning in potential future careers, and exposed them to
+successful women in STEM.
+
+### Learner experiences
+
+We surveyed learners anonymously after each Club and Summer Experience and found
+that they all felt their skills in Python programming,
+problem solving, critical thinking, and collaboration had improved.
+Furthermore, for the 2019 instance of Club we asked that all participants take a
+pre-test and a post-test comprising Python coding questions.
+While only five participants recorded their post-test, all of them answered more
+questions correctly on the post-test than the pre-test (range 1-8 more questions
+correct out of 11).
+
+Overwhelmingly, learners' favorite parts of the program are the guest speakers
+and the project.
+These aspects of our curriculum expose them to new fields and allow them to
+apply their newfound coding skills to asking an interesting question.
```

Comment:
A 2021 Club learner shared, "I plan to go to college for Computer Science and get a robotics minor when my college offers it. GWC has inspired me to consider pursuing a Masters or PhD in CS as well as take some electives in Data Science.” Five of our 86 alumni have gone on to perform research with U-M faculty members, with one presenting her work at an international conference.